lib/sources.js

```diff
@@ -53,18 +47,7 @@ exports.getConfigAliasesSync = function(featureName) {
 };
 
 exports.streamPolyfillSource = function (featureName, type) {
-	const key = featureName + type;
```

Comment:
_(This caused a lot of needless garbage-collection pausing due to large polyfill strings in memory.)_

This was added because we had lots of requests hitting our server, which was causing a lot of filesystem access. With the long surrogate-control (cdn caching), we should no longer need to have an in-memory cache of polyfills.